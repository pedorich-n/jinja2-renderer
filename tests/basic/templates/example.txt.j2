{% from "macros/example_macros.j2" import hello %}
Hello {{ name }}, from {{ program }}
{{ hello("macros") }}
