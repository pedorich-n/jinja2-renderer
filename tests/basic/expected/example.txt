Hello world, from Nix
Hello macros
